Formula/sourcery.rb

```diff
@@ -0,0 +1,18 @@
+class Sourcery < Formula
+  desc "Meta-programming for Swift, stop writing boilerplate code."
+  homepage "https://github.com/krzysztofzablocki/Sourcery"
+  url "https://github.com/krzysztofzablocki/Sourcery/archive/0.5.3.tar.gz"
+  sha256 "ad34e2146e73ce81894ac1dd9130fb2fd12c8213789aef7c5cde9580f813c823"
+
+  depends_on :xcode => ["8.0", :build]
+
+  def install
+    system "unset CC; swift build -c release"
```

Comment:
`swift build -c release` failed with `swift-build: error: invalid inferred toolchain: could not find `clang` at expected path /private/tmp/sourcery-20170125-20504-188t29n/Sourcery-0.5.3/clang`

Is there a different workaround? (Running `xcrun swift build -c release` failed, too)